assets/prototype/application/services/toaster/useLoadingToast.tsx

```diff
@@ -0,0 +1,34 @@
+import find from 'ramda/src/find';
+import React, { useCallback } from 'react';
+import { IToastProps } from '@blueprintjs/core';
+
+import LoadingToastNotice from './LoadingToast';
+import { LoadingToast, LoadingToastCallback } from './types';
+import useDismissToast from './useDismissToast';
+
+const useLoadingToast: LoadingToast = (toaster, hash) => {
+	const dismiss = useDismissToast(toaster, hash);
+	return useCallback<LoadingToastCallback>(
+		(loading = true, message = 'loading', timeout = 0, action = {}, onDismiss = null) => {
+			const isToasting = (msgHash: string): boolean => {
```

Comment:
`isToasting` :smiley:

Reply:
ya I had fun with all of the toast related names